### ISSUE TITLE
Ambient Weather app doesn't work with certain weather stations

### ISSUE THREAD

**Issue by siverson702** [tidbyt/community#990]
I have found that the Ambient Weather app doesn't work if you have a weather station that doesn't support wind speed or UV.   It also doesn't work if the station has multiple weather sensors.   Could we add the option to not show wind/uv and also select which weather station(s) that you'd like to display?  I was able to test this with pixlet by commenting out some sections but I'm not good enough at coding to make a contribution.

**Comment by rs7q5** [comment]
For future reference, it helps to cc the original author for visibility. The author of this app appears to be @maddox.

**Comment by maddox** [comment]
So, it works, it's just that it displays bad?

**Comment by rs7q5** [comment]
Not sure, but from their comment it may error out if it can't get wind speed or UV data? Not sure if you have error checks for missing data.

**Comment by maddox** [comment]
You could pay me $1,000,000 and I couldn't remember 🤣 

I'll have to bring it all up and check it out. Time to finally bust out that spare Tidbyt I picked up for sale!

**Comment by siverson702** [comment]
@maddox  I appreciate the reply!  It doesn't work at all for me.  This is the error I get when I run it in pixlet:

`Error: key "tempf" not in dict`

When I connect to the Ambient Weather API my temps show up as temp1f and temp2f.   I updated the code to change tempf to tempf1 and then it complained about humidity not being in dict.  After changing that to humidity1 it complained about windspeedmph so I commented all the windspeed stuff out.  Then the same for UV and it finally worked.  I just don't know how to code well enough to do much beyond that testing.  Here's what my AW API shows for reference:

`{"dateutc":1674244920000,"tempinf":67.2,"humidityin":25,"battout":1,"temp1f":47.4,"humidity1":40,"batt1":1,"temp2f":44.7,"batt2":1,"feelsLike1":47.4,"dewPoint1":24.3,"feelsLikein":67.2,"dewPointin":30.2,"tz":"America/Los_Angeles","date":"2023-01-20T20:02:00.000Z"},"info":{"name":"Home","coords":{"coords":{"lon":-xxx,"lat":xxx},"address":"My Address, USA","location":"My City","elevation":746.950439453125,"geo":{"type":"Point","coordinates":[-xxx,xxx]}}}}]`
